cloudscale roles with an example project (#48)
cloudscale-ch/ansible-collection-cloudscale: example/README.md

# README

> This is an example ansible project which shows how to build a multi zone setup on cloudscale.ch with two logical cloud environments: _prod_ and _stage_. Each zone may (or may not) use a different cloudscale.ch API token stored in a _secrets.yml_ (which is meant to be encrypted).

Reviewer: ```suggestion
This is an Ansible example project which shows how to build a multi zone setup on cloudscale.ch with two logical cloud environments: _prod_ and _stage_. Each zone may use a different cloudscale.ch API token stored in a _secrets.yml_ (which is meant to be encrypted).
```

> In our project, we defined a bunch of web and databases servers which will be grouped into an _server group_ for anti-affinity.

Reviewer: ```suggestion
In our project, we defined a bunch of web and database servers which will be grouped into a _server group_ for anti-affinity.
```

> This setup could be easily extended to more cloud environments or even used as a green/blue deployment setup.

Reviewer: ```suggestion
This setup could easily be extended to more cloud environments or even used as a green/blue deployment setup.
```

## Setup

```shell

> python3 -m venv .venv

Reviewer: ```suggestion
python3 -m venv venv
```

Otherwise the next line does not work.
